src/app/search/search.component.ts

```diff
@@ -28,8 +28,19 @@ import { ELASTIC_SEARCH_CLIENT } from './elastic-search-client';
 import { QueryBuilderService } from './query-builder.service';
 import { SearchQuery } from './state/search.query';
 import { SearchService } from './state/search.service';
-import { AlertService } from '../shared/alert/state/alert.service';
 
+/**
+ * There are a total of 6 calls per search.
```

Comment:
I always wondered why I saw so many calls in Network Inspector. :)

Reply:
Ya, it's always annoyed me that we have to do 2 calls for the verified/non-verified buckets because the index is not as expected.